Makefile

```diff
@@ -51,8 +54,9 @@ uninstall:
 help:
 	@echo
 	@echo Build commands:
-	@echo "  make [all]     - Build all targets"
-	@echo "  make dgraph    - Build only dgraph binary"
+	@echo "  make [all]     - Build all targets [EE]"
+	@echo "  make oss       - Build all targets [OSS]"
+	@echo "  make dgraph    - Build dgraph binary"
```

Comment:
Any reason we have both `make all` and `make dgraph`? It seems they are the same.